### PR TITLE
PATCH should return 200 OK

### PR DESCRIPTION
**Pull request by solankisamir**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Contribution checklist:
- [x] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of “no breaking changes
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [ ] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [x] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  - Adding a new service

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from API Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable version with Breaking Change Validation errors
- [ ] Updating API(s) in preview over 1 year

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

File: specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2020-06-01-preview/apimcontenttypes.json
Comment (majastrz) [inline specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2020-06-01-preview/apimcontenttypes.json:83]: >head [](start = 7, length = 4)

This looks like a potentially breaking change as per [https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Do you have a breaking change approval for it or can you explain why it's not a breaking change?

File: specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2020-06-01-preview/apimloggers.json
Comment (majastrz) [inline specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2020-06-01-preview/apimloggers.json:348]: >name [](start = 13, length = 4)

Another potential breaking change?

File: specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2020-06-01-preview/apimcontenttypes.json
Comment (solankisamir) [reply specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2020-06-01-preview/apimcontenttypes.json:83]: This is a new api-version and not published yet.

File: specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2020-06-01-preview/apimloggers.json
Comment (solankisamir) [reply specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2020-06-01-preview/apimloggers.json:348]: @majastrz This is a new api-version and not published yet.

File: specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2020-06-01-preview/apimcontenttypes.json
Comment (majastrz) [reply specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2020-06-01-preview/apimcontenttypes.json:83]: Read the link I sent. Yes, breaking changes require an new API version, which you are doing but removing things across API versions is also considered a breaking change and requires separate approval (instructions in the link).

---
In reply to: [466675140](https://github.com/Azure/azure-rest-api-specs/pull/10328#discussion_r466675140) [](ancestors = 466675140)

**Comment by solankisamir** [comment]
@majastrz can you take a look again from ARM side? I have added the reason for breaking change. it is a new api-version we are working on. Not live yet.

**Comment by majastrz** [comment]
Yes, you're right I forgot that this is going into a dev branch instead of master. Not a breaking change.

**Review by majastrz — APPROVED** [review]
Signed off from ARM side.

**Comment by xccc-msft** [comment]
@promoisha Can you please help review from service end? I found your name on CODEOWNERS. ARM already signed off. Thanks.

**Review by promoisha — APPROVED** [review]

**Comment by solankisamir** [comment]
@xccc-msft can you take a look again?

**Pull request merged**
